src/bentoml/_internal/cloud/deployment.py

```diff
@@ -52,6 +119,80 @@ def create_deployment(
             raise BentoMLException("Create deployment request failed")
         return res
 
+    def create_deployment(
+        self,
+        deployment_name: str,
+        bento_repository: Tag,
+        runner_config: dict[str, DeploymentConfig],
+        enable_ingress: bool,
+        mode="function",
+        type=DeploymentTargetType.STABLE,
+        context: str | None = None,
+        clusterName: str | None = None,
+        kube_namespace: str | None = None,
+        api_server_config: DeploymentConfig | None = None,
+        bento_request_overrides: dict[str, t.Any] | None = None,
+        do_not_deploy: bool = False,
+        description: str = "",
+        labels: t.Dict[str, str] | None = None,
+    ):
+        dct = {}
+        for key, one_runner_config in runner_config.items():
+            dct[key] = DeploymentTargetRunnerConfig(
+                hpa_conf=one_runner_config.hpa_conf,
+                resources=one_runner_config.resources,
+                resource_instance=one_runner_config.resource_instance,
+                envs=one_runner_config.envs,
+                enable_stealing_traffic_debug_mode=one_runner_config.enable_stealing_traffic_debug_mode,
+                enable_debug_mode=one_runner_config.enable_debug_mode,
+                enable_debug_pod_receive_production_traffic=one_runner_config.enable_debug_pod_receive_production_traffic,
+                deployment_strategy=one_runner_config.deployment_strategy,
+                bento_deployment_overrides=one_runner_config.bento_deployment_overrides,
+                traffic_control=one_runner_config.traffic_control,
+                deployment_cold_start_wait_timeout=one_runner_config.deployment_cold_start_wait_timeout,
+            )
```

Comment:
no need to do this, we can write custom structure and unstructure strategy for DeploymentTargetRunnerConfig